black.py

```diff
@@ -629,7 +629,7 @@ def format_file_in_place(
     `mode` and `fast` options are passed to :func:`format_file_contents`.
     """
     if src.suffix == ".pyi":
-        mode = evolve(mode, is_pyi=True)
+        mode = mode.replace(is_pyi=True)
```

Comment:
it's actually `dataclasses.replace`, a global function (presumably so dataclasses don't inject a `.replace` method everywhere)